### ISSUE TITLE
wrong params given to layers in squeeze_excitation

### ISSUE THREAD

**Issue by wranai** [KichangKim/DeepDanbooru#55]
This here for [`squeeze_excitation`](https://github.com/KichangKim/DeepDanbooru/blob/50bda534c79294204c846e3c4f11e98a4c127b91/deepdanbooru/model/layers/__init__.py#L87-L92) seems to be incorrect: `s` is just getting overwritten in each line since the layers work on `x`.

```
    s = x


    s = tf.keras.layers.GlobalAveragePooling2D()(s)
    s = tf.keras.layers.Dense(output_filters // reduction, activation="relu")(x)
    s = tf.keras.layers.Dense(output_filters, activation="sigmoid")(x)
    x = tf.keras.layers.Multiply()([x, s])

**Comment by KichangKim** [comment]
Thanks for report. Fixed by 1f6774dd9a7a89d042fb0ae2940e8f12ba845b54
